sdk/core/azure-core/src/main/java/com/azure/android/core/util/Base64Util.java

```diff
@@ -9,78 +9,76 @@
 /**
  * Utility for Base64 encoding and decoding.
  */
-public final class Base64Util {
+public interface Base64Util {
```

Comment:
We shouldn't turn this to an interface given there is no use-case to implement it.

Reply:
This change was made to keep consistence with our CoreUtils implementation, which was decided to be an interface with only static methods as outlined in [this comment](https://github.com/Azure/azure-sdk-for-android/pull/128#discussion_r350471301).